images/linux/scripts/installers/boost.sh

```diff
@@ -7,17 +7,19 @@
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/document.sh
 
-BOOST_ZIP_PATH=/opt/hostedtoolcache/Boost
 BOOST_LIB=/usr/local/share/boost
 
 # Install Boost
 for BOOST_VERSION in ${BOOST_VERSIONS//,/ }
 do
     BOOST_SYMLINK_VER=`echo "${BOOST_VERSION//[.]/_}"`
     BOOST_ROOT="BOOST_ROOT_$BOOST_SYMLINK_VER"
-    BOOST_ZIP="boost_`echo $BOOST_VERSION`_gcc.zip"
 
-    unzip $BOOST_ZIP_PATH/$BOOST_ZIP -d $BOOST_LIB
+    wget -q https://dl.bintray.com/boostorg/release/${BOOST_VERSION}/source/boost_${BOOST_SYMLINK_VER}.tar.bz2
+    tar -xf boost_${BOOST_SYMLINK_VER}.tar.bz2
+    cd boost_${BOOST_SYMLINK_VER}
+    ./bootstrap.sh --prefix=$BOOST_LIB/$BOOST_VERSION
+    ./b2 install
```

Comment:
Previously, version 1.69 was built with the following options:
```
sudo ./b2 install --prefix=$BOOST_DIR --build-dir=$BOOST_TEMP_DIR --layout="tagged" variant="release" link="static,shared" runtime-link="static,shared"
sudo ./b2 install --prefix=$BOOST_DIR --build-dir=$BOOST_TEMP_DIR --layout="tagged" variant="debug" link="static,shared" runtime-link="static,shared"
```

But if we do that during image-generation, it will add about 30-50 minutes for each Boost version.